src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/Resource.sql

```diff
@@ -5,16 +5,13 @@
     Version                     int                     NOT NULL,
     IsHistory                   bit                     NOT NULL,
     ResourceSurrogateId         bigint                  NOT NULL,
-    CONSTRAINT UQ_Resource_ResourceSurrogateId UNIQUE (ResourceSurrogateId)
-    ON [Primary],
-    CONSTRAINT PKC_Resource PRIMARY KEY CLUSTERED (ResourceTypeId, ResourceSurrogateId)
-    WITH (DATA_COMPRESSION = PAGE)
-    ON PartitionScheme_ResourceTypeId(ResourceTypeId),
     IsDeleted                   bit                     NOT NULL,
     RequestMethod               varchar(10)             NULL,
     RawResource                 varbinary(max)          NOT NULL,
     IsRawResourceMetaSet        bit                     NOT NULL DEFAULT 0,
-    SearchParamHash             varchar(64)             NULL
+    SearchParamHash             varchar(64)             NULL,
```

Comment:
>, [](http://example.com/codeflow?start=60&length=1)

Remove last comma

Reply:
The comma here can't be removed, as the generated 26.sql appends next line(primary key constraint) with SearchParamHash